lib/tasks/downloads.rake

```diff
@@ -41,19 +41,23 @@ task :downloads => [:environment, :windows_downloads, :mac_downloads]
 
 task :windows_downloads => :environment do
   # find latest windows version
-  project = "msysgit"
-  win_downloads = file_downloads_from_github("msysgit/msysgit")
+  project = "git-for-windows"
+  win_downloads = file_downloads_from_github("git-for-windows/git")
   win_downloads.each do |url, date|
     name = url.split('/').last
-    if m = /^Git-(.*?)-(.*?)(\d{4})(\d{2})(\d{2})\.exe/.match(name)
-      version = m[1]
+    if m = /^(.*?)Git-(.*?)-(.*?)-(.*?)-(.*?)-bit\.(.*?)exe/.match(name)
+      portable = m[1]
+      version = m[2]
+      bitness = m[5]
+      puts portable
       puts version
+      puts bitness
       puts name
       puts url
       puts date
       puts
-      if v = Version.where(name: version).first
-        options = {version: v, url: url, filename: name, platform: 'windows', release_date: date}
+      if v = Version.where(name: version[0..-3]).first
```

Comment:
This will cause problems if there's ever a release of the form 2.3.5.10.  It might be better to replace this with one of the following:
```ruby
if v = Version.where(name: version.slice(/\d*\.\d*\.\d*/)).first
if v = Version.where(name: version[/\d*\.\d*\.\d*/]).first
```